features/visualization/pptx/slide_editor.py

```
@@ -0,0 +1,672 @@
+    디자이너 서식을 보존하면서 텍스트와 차트 데이터만 교체하는 편집기.
+
+    식별자는 PowerPoint 가 자동 부여한 `cNvPr/@id` 만 사용한다.
+    """
+
+    PML_NS = "http://schemas.openxmlformats.org/presentationml/2006/main"
+    DRAWINGML_NS = "http://schemas.openxmlformats.org/drawingml/2006/main"
+    CHART_NS = "http://schemas.openxmlformats.org/drawingml/2006/chart"
+    REL_NS = "http://schemas.openxmlformats.org/officeDocument/2006/relationships"
+    PKG_REL_NS = "http://schemas.openxmlformats.org/package/2006/relationships"
+
+    _TITLE_PLACEHOLDER_TYPES = {"title", "ctrTitle", "subTitle"}
```

Ruff (0.15.13)

[warning] Mutable default value for class attribute

(RUF012)